### PR TITLE
fix(docs): Fixed Local Filesystem doc

### PR DESCRIPTION
**Pull request by vasd85**

### CODE REVIEW

File: docs/build/local-data-storage.md [hunk 1]
```diff
@@ -6,60 +6,64 @@ A peer hosting services may agree to provide access to their filesystem or alloc
...
+```bash
 fluence init write-to-file
+```
+```
 ? Select template minimal
+? Select Fluence Environment to use by default with this project kras (default)
 
-Successfully initialized Fluence project template at ~/localdev/write-to-file
+Successfully initialized Fluence project template at ~/your-path/write-to-file
+```
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md:36]: that's an incomplete description of the process. we need to present the masks for template and network selection.

File: docs/build/local-data-storage.md [hunk 2]
```diff
@@ -6,60 +6,64 @@ A peer hosting services may agree to provide access to their filesystem or alloc
...
-? Do you want to use services as the name of your new service? No
-? Enter service name (must start with a lowercase letter and contain only letters, numbers, and underscores)
-filesys_adapter
-Successfully generated template for new service at services
-? Do you want add filesys_adapter to fluence.yaml? Yes
+```
+Successfully generated template for new service at ~/your-path/write-to-file/src/services/filesys_adapter
     Updating crates.io index
-
+   Compiling ...
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md]: it's better to be explicit in indicating that lines were cut, e.g.,
<...>

File: docs/build/local-data-storage.md [hunk 3]
```diff
@@ -6,60 +6,64 @@ A peer hosting services may agree to provide access to their filesystem or alloc
...
+   Compiling filesys_adapter v0.1.0 (~/your-path/write-to-file/src/services/filesys_adapter/modules/filesys_adapter)
+    Finished release [optimized] target(s) in 29.35s
+Added filesys_adapter to ~/your-path/write-to-file/fluence.yaml
+? Do you want to add service filesys_adapter to a default deal dealName: Yes
+Added filesys_adapter to dealName
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md:54]: don't cut the prompts please
```
added filesys_adapter to /Users/bebo/localdev/write-to-file/fluence.yaml
? Do you want to add service filesys_adapter to a default deal dealName: (Y/n)
```

File: docs/build/local-data-storage.md [hunk 4]
```diff
@@ -72,26 +76,47 @@ pub fn main() {}
...
+        │       ├── module.yaml
+        │       └── src
+        │           └── main.rs
+        └── service.yaml
+
+10 directories, 8 files
 ```
 
-With the scaffolding out of the way, let's code our modules. First, we create our general-purpose, reusable "adapter" module to handle the put/get operations to/from the local, i.e., host peer, filesystem and add a file removal method, `rm`, as well:
+With the scaffolding out of the way, let's code our modules. First, we create our general-purpose, reusable `use_filesys` module to handle the put/get operations to/from the local, i.e., host peer, filesystem and add a file removal method, `rm`, as well:
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md]: there needs to be an explanation why the modules are placed in different dirs.

```suggestion
With our service and modules scaffolded, we can now proceed and code our (reusable) `use_filesys` module to handle the interaction with the host filesystem.```

File: docs/build/local-data-storage.md [hunk 5]
```diff
@@ -127,64 +152,60 @@ pub fn rm(name: String) -> String {
         Ok(_) => "OK".to_string(),
         Err(e) => e.to_string(),
     }
-}```
+}
+```
 
 The code is pretty straight forward read/write but a few lines are noteworthy: The `const SITES_DIR` is the *alias* to the local path and needs to match up with the mapping in `modules.yaml`.
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md]:
```suggestion
The code is pretty straight forward but note `const SITES_DIR`:  This is the **alias** to the local path and needs to match up with the `volumes` mapping in `modules.yaml`, which we still need to add.
```

File: docs/build/local-data-storage.md [hunk 6]
```diff
@@ -127,64 +152,60 @@ pub fn rm(name: String) -> String {
...
 ```yaml
 version: 0
 type: rust
-name: filesys_adapter
-
+name: use_filesys
 ```
 
-Which is not enough. We need to specify that we want this module to be able to access the hosts filesystem via some specified directory and we do this my mounting a named directory mapped to the SITES_DIR alias:
+Which is not enough. We need to specify that we want this module to be able to access the hosts filesystem via some specified directory, and we do this by mounting a named directory mapped to the SITES_DIR alias:
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md]:
```suggestion
As mentioned earlier, we need to provide a reference to the location of the file directory on the host system. We do this by referencing the `.tmp`, which needs to be available on the host system. If the specified dir is not available on the host system, an error will occur at runtime.
By specifying the `sites` directory, we are enabling the mounting the named directory mapped to the SITES_DIR alias in our rust code.
```

File: docs/build/local-data-storage.md [hunk 7]
```diff
@@ -127,64 +152,60 @@ pub fn rm(name: String) -> String {
...
 type: rust
-name: filesys_adapter
+name: use_filesys
 volumes:
   sites: ./tmp
-
 ```
 
-For more information on import functions see the [Marine book](/docs/marine-book/marine-runtime/configuration-file) and the [configuration properties](https://github.com/fluencelabs/fluence-cli/tree/main/docs/configs/module.md) provided by Fluence CLI.
+Where the *volumes* section contains the mapping of the actual directory to the alias we set in the `module.yaml`. For more information on import functions see the [Marine book](/docs/marine-book/marine-runtime/configuration-file) and the [configuration properties](https://github.com/fluencelabs/fluence-cli/tree/main/docs/configs/module.md) provided by Fluence CLI.
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md]:
```suggestion
For more information on import functions see the [Marine book](/docs/marine-book/marine-runtime/configuration-file) and the [configuration properties](https://github.com/fluencelabs/fluence-cli/tree/main/docs/configs/module.md) provided by Fluence CLI.
```

File: docs/build/local-data-storage.md [hunk 8]
```diff
@@ -256,82 +277,77 @@ pub fn rm_file(name: String) -> IOResult {
...
+```
+```
+# Making sure all services are downloaded...
+# Making sure all services are built...
+   Compiling filesys_adapter v0.1.0 (~/your-path/write-to-file/src/services/filesys_adapter/modules/filesys_adapter)
+    Finished release [optimized] target(s) in 1.79s
 ```
 
-All looks good and we now have two Wasm module we'd like to use as a service. See the respective `target\\wasm32-wasi\\release` directories for the *.wasm files.
+All looks good, and we now have two Wasm module we'd like to use as a service. See the respective `target/wasm32-wasi/release` directories for the *.wasm files.
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md]:
```suggestion
All looks good and we now have two Wasm module we'd like to use as a service. See the respective `target/wasm32-wasi/release` directories for the *.wasm files.
```
(closely connected, short)

File: docs/build/local-data-storage.md [hunk 9]
```diff
@@ -346,36 +362,47 @@ data IOResult:
...
+}
+ elapsed time: 3.504709ms
 
-6>
+5> call filesys_adapter read_file ["test-file.txt"]
+result: {
+  "stderr": "error reading file",
+  "stdout": ""
+}
+ elapsed time: 642.292µs
```
Comment (boneyard93501) [inline docs/build/local-data-storage.md:436]: what about the section below? 
should remove the worker deploy references, deploy to kras and add/run tests maybe?

**Review by boneyard93501 — DISMISSED** [review]

File: docs/build/local-data-storage.md (hunk 3 above)
Comment (vasd85) [reply docs/build/local-data-storage.md:38]: Do you mean to show initial prompt and than the answer, like this?
```bash
vasd85@MacBook-Pro write-to-file % fluence service new filesys_adapter
Successfully generated template for new service at /Users/vasd85/fluence/test/write-to-file/src/services/filesys_adapter
Added filesys_adapter to /Users/vasd85/fluence/test/write-to-file/fluence.yaml
? Do you want to add service filesys_adapter to a default deal dealName: (Y/n)

```

```bash
? Do you want to add service filesys_adapter to a default deal dealName: Yes
Added filesys_adapter to dealName
    Updating crates.io index
   Compiling proc-macro2 v1.0.76
   <...>
   Compiling filesys_adapter v0.1.0 (/Users/vasd85/fluence/test/write-to-file/src/services/filesys_adapter/modules/filesys_adapter)
    Finished release [optimized] target(s) in 29.95s
vasd85@MacBook-Pro write-to-file %
```

File: docs/build/local-data-storage.md [hunk 10]
```diff
@@ -6,60 +6,85 @@ A peer hosting services may agree to provide access to their filesystem or alloc
...
-The [accompanying repo](notion://www.notion.so/fluencenetwork/Fluence-Developer-Documentation-bdf8d06ad52e493fb765456dbd5480cd).
-
 Let's start by creating a new, minimal Fluence CLI project by the name of your choice:
 
-```
+```bash
 fluence init write-to-file
+```
+```
+vasd85@MacBook-Pro test % fluence init write-to-file
```
Comment (shamsartem) [inline docs/build/local-data-storage.md]: Here and in other places I would suggest to remove your personal terminal prefix from docs, just replace it with $ or maybe better to remove it completely and leave only the relevant part which is the command itself

**Review by boneyard93501 — APPROVED** [review]

**Pull request merged**